### PR TITLE
switch website to docusaurus

### PR DESCRIPTION
**Pull request by 2pk03**

### CODE REVIEW

**Review by suyanhanx — COMMENTED** [review]
Hi, I checked the files you committed.
But in fact, you should not submit the `.docusaurus` and `node_modules` folder. These two folders are related to development only. You are not supposed to commit to the code repository. 
They should be configured by .gitignore.

Maybe you could check our workflow as an example? 
https://github.com/apache/opendal/blob/e94f068a9977fcd20c9bb3626aaaa8be833209a8/.github/workflows/docs.yml#L381-L394

**Comment by 2pk03** [comment]
We need to rebase main, since the whole structure changed. I did not implement the research papers, since they aren't directly connected to Wayang. For the latest paper we should write a blogpost.

**Comment by 2pk03** [comment]
> Hi, I checked the files you committed. But in fact, you should not submit the `.docusaurus` and `node_modules` folder. These two folders are related to development only. You are not supposed to commit to the code repository. They should be configured by .gitignore.
> 
> Maybe you could check our workflow as an example? https://github.com/apache/opendal/blob/e94f068a9977fcd20c9bb3626aaaa8be833209a8/.github/workflows/docs.yml#L381-L394

Ya just saw it ...thanks for pointing out :)

File: yarn.lock
Comment (suyanhanx) [inline yarn.lock:1]: You submitted two lock files, `yarn.lock` and `package-lock.json`.
You can choose one of them.

File: .asf.yaml [hunk 1]
```diff
@@ -1,48 +1,43 @@
...
-    merge:   true
-    # disable rebase button:
+    merge:   false
     rebase:  false
+  protected_branches:
+    main: {}
+    deploy: {}
+
+publish:
+  whoami: deploy
```
Comment (glauesppen) [inline .asf.yaml:36]: Are we going to use this branch to put the site into production?

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+          node-version: 18.x
+      - run: npm install
+      - run: npm run build
+      - run: cp .asf.yaml build/.asf.yaml
+      - uses: peaceiris/actions-gh-pages@v3
+        if: github.event_name != 'pull_request'
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./build
+          publish_branch: deploy
```
Comment (glauesppen) [inline .github/workflows/deploy.yml:25]: Same thing here about the branching. 
Shouldn't we keep the asf-site as the production branch?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,29 @@
-# Wayang Website
-Apache Wayang is the API for big data and data processing across multiple data processing frameworks. Wayang enables data engineers, data scientists and data operations to use multiple platforms at the same time. The repo contains the website for [Apache Wayang (incubating)](https://wayang.apache.org)
+# Apache Fury (incubating) Website
```
Comment (glauesppen) [inline README.md:1]: It's referencing the Fury website

File: blog/2024-01-25-website_update.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+We're updated our website and use now Docusaurus.
+
+<!--truncate-->
+# Website updated
+
+Author: [2pk03](https://github.com/2pk03)
+
+We switched to a new CMS. That's all.
+
+## Cheatsheet
```
Comment (glauesppen) [inline blog/2024-01-25-website_update.md:17]: This is an example of the Docusaurus. I think it should be deleted

**Review by glauesppen — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (2pk03) [reply .github/workflows/deploy.yml:25]: good point, +1

File: yarn.lock
Comment (2pk03) [reply yarn.lock:1]: Ya figured too, was editing more content ;) Ty!

File: blog/2024-01-25-website_update.md (hunk 4 above)
Comment (2pk03) [reply blog/2024-01-25-website_update.md:17]: Na, I did that to have a blog entry :)

**Comment by 2pk03** [comment]
Also created a 2nd PR against website-new, to make a switch more easy. We can close one of the PR's, was just to make it more easy.

File: README.md (hunk 3 above)
Comment (2pk03) [reply README.md]: hmm, its not in my local repo. I close this PR since it's too messed, the other one is a clean sync into an orphan branch (docusaurus).

**Comment by 2pk03** [comment]
It's too confusing right now, created a new PR into an orphan branch (docusaurus).

**Pull request closed without merging**